Spigot-Server-Patches/0551-Tweaked-Patrol-Spawning.patch

```diff
@@ -0,0 +1,42 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheShermanTanker <tanksherman27@gmail.com>
+Date: Sat, 8 Aug 2020 02:14:48 +0800
+Subject: [PATCH] Tweaked-Patrol-Spawning
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index e471e764935e2a89560de56959a782b02e5e8fe1..84d6dee81610a727a52026b80a5757ae1ab2ba23 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -578,14 +578,12 @@ public class PaperWorldConfig {
+         generateFlatBedrock = getBoolean("generator-settings.flat-bedrock", false);
+     }
+ 
+-    public boolean disablePillagerPatrols = false;
```

Comment:
Check the contributing.md file for how to merge this into the file which creates it.

Reply:
I did but the merge is not working :/